src/Impostor.Client/Impostor.Client.csproj

```diff
@@ -1,102 +1,28 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I guess having the new csproj is great but why change the target framework though?

Reply:
`System.Resources.Extensions` (https://github.com/dotnet/msbuild/issues/4704) requires netstandard2.0 (net461)